build.sh

```diff
@@ -55,13 +60,22 @@ if [ -e platforms/${PLATFORM}/set-env.sh ]; then
     . platforms/${PLATFORM}/set-env.sh
 fi
 
+if [ ${#STD[@]} -eq 0 ]; then
+    STD+=("11")
+fi
+
+STD=($(printf "%s\n" "${STD[@]}" | uniq ))
+
+for i in "${STD[@]}"
+do
 # generate make file
-${STRACE_CMD} cmake ${PRC_CMAKE_EXTRA_FLAGS} -B${BUILD_DIR} -H. "-DCMAKE_BUILD_TYPE=${BUILD_TYPE}" ${TOOLCHAIN}
+${STRACE_CMD} cmake ${PRC_CMAKE_EXTRA_FLAGS} -B"cpp${i}/${BUILD_DIR}" -H. "-DCMAKE_BUILD_TYPE=${BUILD_TYPE}" ${TOOLCHAIN} "-DCMAKE_STD=${i}"
```

Comment:
I guess variables prefixed CMAKE_ are for use by cmake, so let's have our own prefix which isn't confused with cmake